cloudos/__main__.py

```diff
@@ -11,6 +11,7 @@
 # GLOBAL VARS
 JOB_SUCCESS = 'completed'
 JOB_FAIL = 'failed'
```

Comment:
for consistency, should we also modify the status `JOB_FAIL` to `JOB_FAILED` and also `JOB_SUCCESS` to `JOB_COMPLETED`?